tutorials/10-bayesian-differential-equations/10_bayesian-differential-equations.jmd

```diff
@@ -73,6 +73,63 @@ To make the example more realistic we add random normally distributed noise to t
 sol = solve(prob, Tsit5(); saveat=0.1)
 odedata = Array(sol) + 0.8 * randn(size(Array(sol)))
 
+# alternatively, we can use real-world data from Hudson’s Bay Company records (a Stan implementation can be found here: https://mc-stan.org/users/documentation/case-studies/lotka-volterra-predator-prey.html)
```

Comment:
As it is written currently, it is not an alternative: It is used instead of the simulated data. So the explanation above would have to be fixed and the code that generates the noise simulated data could be removed.

Reply:
@devmotion yes it's replacing the simulated data. Maybe we can keep both?

Reply:
IMO that's just confusing. I think either we present both approaches fully or we just remove the dead code. I think I'm in favour of the latter, to keep the size of the tutorial and its execution times manageable.

Note though that Stan uses a different model for the Hudson Bay dataset (different priors etc.). So I'm not sure at all that the different dataset will produce useful results at all currently (or rather I assume it doesn't).

Reply:
@devmotion Thanks David for offering the thoughts, yes, I just re-removed the real-world data because:
1. leaving the choice to users may be confusing;
2. The Stan model is a bit different from ours here. 
as you mentioned in previous comments.